### ISSUE TITLE
Incompatible with vue.js

### ISSUE THREAD

**Issue by DCRichards** [webpack-contrib/stylelint-webpack-plugin#178]
* Operating System: OSX 10.14.5
* Node Version: `node:12-alpine (docker)`
* NPM Version: `node:12-alpine (docker)`
* webpack Version: `vue 2.6.10`
* stylelint-webpack-plugin Version: `0.10.5`

This issue is for a:

- [x] **bug**
- [ ] **feature** request
- [ ] **modification** request

##### vue.config.js

```js
const StylelintPlugin = require('stylelint-webpack-plugin');

module.exports = {
  lintOnSave: true,

  configureWebpack: {
    plugins: [
      new StylelintPlugin({
        failOnError: true,
        emitErrors: true,
        files: '**/*.(vue|scss|css|html)',
        syntax: 'scss',
      }),
    ],
  },
};
```

### Expected Behavior

Webpack should fail on `yarn serve` and report specific error.

### Actual Behavior

`Starting development server...` is shown and it hangs indefinitely, suggesting it is not being sent to the webpack error handler.

### How Do We Reproduce?

1. Create a new vuejs project `vue create .`, I'm using docker but don't believe this is related.
2. `yarn add --dev stylelint stylelint-webpack-plugin`
3. Set the above as you `vue.config.js`
4. Add some bad SCSS (empty block for example)
5. `yarn serve`
6. Dev server hangs with no errors.

**Comment by ricardogobbosouza** [comment]
@DCRichards it seems to be a problem with `@vue/cli` that does not issue errors on startup.
This test proves that: 
```js
// vue.config.js
class ExamplePlugin {
    apply(compiler) {
      compiler.hooks.watchRun.tapAsync('ExamplePlugin', (compilation, callback) => {
        callback(new Error('foo'))
      });
    }
}

module.exports = {
  configureWebpack: {
    plugins: [
      new ExamplePlugin(),
    ],
  },
};
```
